coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -889,7 +889,7 @@ def linear(context, node):
     context.add(res, torch_name=node.name)
 
 
-@register_torch_op(torch_alias=["conv2d"])
+@register_torch_op(torch_alias=["conv1d", "conv2d", "conv3d", "conv_transpose1d", "conv_transpose2d", "conv_transpose3d"])
```

Comment:
Since we already have passing unit tests for `conv1d`, I don't think we should add an alias for it here. PyTorch must be lowering it to other ops which we do support.

Reply:
Can you help me then understand why I get ?

```
the following model ops are IMPLEMENTED:
  add
  atan2
  clamp
  complex
  constant
  constantchunk
  cos
  exp
  gelu
  layer_norm
  linear
  mul
  sin
  tensor
  transpose
the following model ops are MISSING:
  conv1d
  floatimplicit
  istft
Traceback (most recent call last):
  File "/home/aalvarez/Projects/main/apps/tts-train/vocoder/export.py", line 73, in <module>
    main()
  File "/home/aalvarez/Projects/main/apps/tts-train/vocoder/export.py", line 54, in main
    model.to_coreml(
  File "/home/aalvarez/.virtualenvs/tts-train-XZ1ykfT_-py3.9/lib/python3.9/site-packages/torch/utils/_contextlib.py", line 115, in decorate_context
    return func(*args, **kwargs)
  File "/home/aalvarez/Projects/main/apps/tts-train/vocoder/vocos/__init__.py", line 502, in to_coreml
    coreml_mdl = ct.convert(
  File "/home/aalvarez/Projects/coremltools/coremltools/converters/_converters_entry.py", line 553, in convert
    mlmodel = mil_convert(
  File "/home/aalvarez/Projects/coremltools/coremltools/converters/mil/converter.py", line 188, in mil_convert
    return _mil_convert(model, convert_from, convert_to, ConverterRegistry, MLModel, compute_units, **kwargs)
  File "/home/aalvarez/Projects/coremltools/coremltools/converters/mil/converter.py", line 212, in _mil_convert
    proto, mil_program = mil_convert_to_proto(
  File "/home/aalvarez/Projects/coremltools/coremltools/converters/mil/converter.py", line 286, in mil_convert_to_proto
    prog = frontend_converter(model, **kwargs)
  File "/home/aalvarez/Projects/coremltools/coremltools/converters/mil/converter.py", line 108, in __call__
    return load(*args, **kwargs)
  File "/home/aalvarez/Projects/coremltools/coremltools/converters/mil/frontend/torch/load.py", line 75, in load
    return _perform_torch_convert(converter, debug)
  File "/home/aalvarez/Projects/coremltools/coremltools/converters/mil/frontend/torch/load.py", line 122, in _perform_torch_convert
    raise e
  File "/home/aalvarez/Projects/coremltools/coremltools/converters/mil/frontend/torch/load.py", line 114, in _perform_torch_convert
    prog = converter.convert()
  File "/home/aalvarez/Projects/coremltools/coremltools/converters/mil/frontend/torch/converter.py", line 484, in convert
    convert_nodes(self.context, self.graph)
  File "/home/aalvarez/Projects/coremltools/coremltools/converters/mil/frontend/torch/ops.py", line 88, in convert_nodes
    raise RuntimeError(
RuntimeError: PyTorch convert function for op 'conv1d' not implemented.
```

Reply:
Are you calling `torch.jit.trace` prior to conversion?

Reply:
No, I'm using `torch.script` only and I guess there lies the discrepancy.

In my experience, using `torch.script` is much more reliable because of:
- data dependent models with conditionals (main point)
- additions of extra functions
- no input example boilerplate